src/Sentry.Unity/UnityEventProcessor.cs

```diff
@@ -8,21 +8,20 @@
 
 namespace Sentry.Unity
 {
+    internal static class UnitySdkInfo
+    {
+        public const string Version = "0.0.1-alpha";
+        public const string Name = "sentry.dotnet.unity";
+        public const string PackageName = "github:sentry.unity";

```

Comment:
https://github.com/getsentry/sentry-release-registry/pull/48/files#diff-fb5dfde27d2f18f59c7038d18ec4184fd0dae88d88075e5e261474fdf09a2461R3

```suggestion
        public const string PackageName = "upm:sentry.unity";

```